Marlin/Configuration_adv.h

```diff
@@ -755,6 +757,28 @@
 // Some clients will have this feature soon. This could make the NO_TIMEOUTS unnecessary.
 //#define ADVANCED_OK
 
+// @section chunk
+
+/**
+ * This is an experimental feature that allows a host device to send
+ * direct stepper buffers (chunks) instead of G0/1 movement commands.
+ * This frees up processing power on the Marlin device and allows
+ * a higher consistent step rate.
+ *
+ * This is useful when used with an "external planner" that
+ * acts as the USB host and controls the GCode stream while
+ * producing and triggering direct stepper chunks using the
+ * described protocol:
+ *
+ * <!insert links to docs here!>
```

Comment:
@thinkyhead do you know where would be a good place to write a draft specification for this support? ill have to dig around the rep-rap wiki and see if theres a good dumping ground somewhere, but that might not be the right place...